.travis.yml

```diff
@@ -9,4 +10,4 @@ before_install:
 - sudo apt-get install dotnet-sdk-2.0.0
 
 script:
-- dotnet test tests/Faithlife.Utility.Tests -c Release /p:SourceLinkCreate=false /p:SourceLinkTest=false
+- ./build.sh --target=NuGetPackage
```

Comment:
I don't know why SourceLink is failing on Travis (seems to work OK in octokit, for example), but I think this MSBuild property hack could be replicated by changing the `"Build"` task as follows:

```
		var buildSettings = new DotNetCoreBuildSettings { Configuration = configuration, ArgumentCustomization = args => args.Append("--verbosity normal") };
		if (Context.Environment.Platform.IsUnix())
		{
			buildSettings.MSBuildSettings = new DotNetCoreMSBuildSettings
			{
				Properties =
				{
					{ "SourceLinkCreate", new[] { "false" } },
					{ "SourceLinkTest", new[] { "false" } },
				}
			};
		}
		DotNetCoreBuild(solutionFileName, buildSettings);
```

That would probably turn it off for macOS too; don't know if that's a concern. (Not sure if/how Cake distinguishes between Linux and macOS.)

Reply:
That's unlikely to fix the `net46`-related errors in the current PR build though.

Reply:
@ejball suggests that sourcelink fails because it [requires netcoreapp1.0](https://github.com/ctaggart/SourceLink/blob/master/dotnet-sourcelink-git/dotnet-sourcelink-git.csproj#L5) and that SDK isn't installed.

We could modify the travis.yml file to install it, or use the "hack" above. Thoughts?

Reply:
Maybe follow these steps: https://codeblog.jonskeet.uk/2017/08/15/using-net-core-2-0-sdk-on-travis/

Reply:
I figured it out on my own, then stumbled across Jon Skeet's post, then your comments here. 😩